NorthstarDLL/mods/modmanager.cpp

```diff
@@ -56,6 +56,14 @@ Mod::Mod(fs::path modDir, char* jsonBuf)
 
 	Name = modJson["Name"].GetString();
 
+	// Don't load blacklisted mods
+	// TODO [Fifty]: Make this better and possibly add a launch arg to disable this check
+	if (Name == "Mod Settings")
```

Comment:
This isn't really expandable in case we wanna add more mods in the future. Black listed mod names should be in a list.

Reply:
Done